### ISSUE TITLE
Add one plain row in Servers config on TP

### ISSUE THREAD

**Issue by tyagian** [apache/trafficcontrol#2764]
Can we add one more row to the servers>[server_hostname] which can be used for adding useful info like server serial number? We do have them on delivery services so adding such space to servers will also be useful.

**Comment by mitchell852** [comment]
I'm not sure what you mean. Are you saying you want to add another field to a Server? Is this a field like server.notes?

**Comment by mitchell852** [comment]
@tyagian ^^

**Comment by tyagian** [comment]
Yes! It's good to have a notes field in servers config. We have a couple of them in delivery services config but servers have None of them.

**Comment by jhg03a** [comment]
Is the new data generally desirable or environment specific?  If it's generally desirable we should consider adding it to our existing data model instead of an unstructured text field.

**Comment by guzzijason** [comment]
FYI, I've run into another use case for this: if we had a field to store something such as serial number or MAC address for a server object in TO, it would facilitate our use of "universal" install images, where the servers bootstrap themselves automatically and get all the info they need to fully configure themselves from TOAPI. Essentially, if this worked, we could potentially do away with geniso.

**Comment by DoctorNantucket** [comment]
@mitchell852, notice the comment above. I'd love to at a minimum have a conversation about what @guzzijason mentioned. If adding this new field would lead to us being able to more easily get our new hardware up and running this should have a higher priority.

**Comment by guzzijason** [comment]
i.e we would love to have new gear pre-imaged with a universal ISO before it is even racked/stacked, and let it install itself. Would save literal man-months of time on these new deploys, IMHO.

**Comment by dneuman64** [comment]
There was a long discussion about this on the dev mailing list that got mostly filibustered:  https://lists.apache.org/thread.html/1af20d2ed98b2d99bf64d2de1bd9c7a9ecd707038083f18cb51a839a@%3Cdev.trafficcontrol.apache.org%3E
It's probably worth trying to bring up again.

**Comment by guzzijason** [comment]
FYI, I've found that ILO IP functions as an effective analog of a serial number, for host self-identification purposes. For now, I'm just going with that for the project I'm working on.
